### ISSUE TITLE
Filtering the list of questions in the Form Builder causes the sidebar width to change drastically

### ISSUE THREAD

**Issue by neel-a-mehta** [civiform/civiform#1226]
Each time you type something in the form builder's filter view, the list of questions shown to you in the sidebar changes. If questions with particularly long names are excluded, then the sidebar becomes narrower. This is usually OK, but it means that the sidebar width changes unpredictably and erratically as the user is typing things into the filter bar.

Before filtering:
![5tzKdQyb7Mi4FZn](https://user-images.githubusercontent.com/77128518/119054451-48552900-b98d-11eb-9352-7edb0a8cbbd9.png)

After (notice different sidebar width):
![6vBRBFk7MwLVnDT](https://user-images.githubusercontent.com/77128518/119054448-47bc9280-b98d-11eb-94ed-3ec5aaefd502.png)

Can we set a sort of fixed width on the sidebar so that it doesn't change as the user types into filter bar?

First noticed after #1224

**Comment by cassandralfernandes** [comment]
@azizova-leyla is this issue related to [#2808](https://github.com/civiform/civiform/issues/2808) (which you are already working on)?

**Comment by azizova-leyla** [comment]
Yup, that's the same root cause. We can close this as duplicate.

**Comment by cassandralfernandes** [comment]
closed as duplicate of issue #2808
